pkg/utils/logutil/log_test.go

```diff
@@ -33,52 +35,172 @@ func TestStringToZapLogLevel(t *testing.T) {
 	re.Equal(zapcore.InfoLevel, StringToZapLogLevel("whatever"))
 }
 
+func TestRedactInfoLogType(t *testing.T) {
+	re := require.New(t)
+	// JSON unmarshal.
+	jsonUnmarshalTestCases := []struct {
+		jsonStr string
+		expect  RedactInfoLogType
+	}{
+		{`false`, RedactInfoLogOFF},
+		{`true`, RedactInfoLogON},
+		{`"MARK"`, RedactInfoLogMark},
+		{`"OTHER"`, RedactInfoLogON},
+		{`"OFF"`, RedactInfoLogOFF},
+		{`"ON"`, RedactInfoLogON},
+		{`"mark"`, RedactInfoLogMark},
+		{`"off"`, RedactInfoLogOFF},
+		{`"on"`, RedactInfoLogON},
+		{`""`, RedactInfoLogOFF},
+		{`"fALSe"`, RedactInfoLogOFF},
+		{`"trUE"`, RedactInfoLogON},
+	}
+	var redactType RedactInfoLogType
+	for _, tc := range jsonUnmarshalTestCases {
+		err := json.Unmarshal([]byte(tc.jsonStr), &redactType)
+		re.NoError(err)
+		re.Equal(tc.expect, redactType)
+	}
+	// JSON marshal.
+	jsonMarshalTestCases := []struct {
+		typ    RedactInfoLogType
+		expect string
+	}{
+		{RedactInfoLogOFF, `false`},
+		{RedactInfoLogON, `true`},
+		{RedactInfoLogMark, `"MARK"`},
+	}
+	for _, tc := range jsonMarshalTestCases {
+		b, err := json.Marshal(tc.typ)
+		re.NoError(err)
+		re.Equal(tc.expect, string(b))
+	}
+	// TOML unmarshal.
+	tomlTestCases := []struct {
+		tomlStr string
+		expect  RedactInfoLogType
+	}{
+		{`redact-info-log = false`, RedactInfoLogOFF},
+		{`redact-info-log = true`, RedactInfoLogON},
+		{`redact-info-log = "MARK"`, RedactInfoLogMark},
```

Comment:
Should we use marker instead of mark?

Reply:
Nice catch. It should be `MARKER` as [TiDB does](https://docs.pingcap.com/tidb/stable/system-variables#tidb_redact_log). I will fix it.